### DISCUSSION TITLE
Errore get_sku() on bool

### DISCUSSION THREAD

**Discussion by logoso** [fattureincloud/api#116] [Q&A] [answered]
Ciao a tutti
sono nuovo qui, ho provato ad installare il plugin di WP ma all'avvio ottengo questo errore:
Call to a member function get_sku() on bool
Qualcuno saprebbe darmi qualche dritta?

Grazie

**Comment by valmoz** [comment, marked as answer]
Buongiornio @logoso, posso chiederle a quale plugin sta facendo riferimento?

**Reply by logoso** [reply]
Fatture in cloud per Wordpress

**Reply by valmoz** [reply]
Per favore, le chiederei di incollare il link alla pagina del plugin su questa discussion.

Grazie
Mauro

**Reply by logoso** [reply]
eccolo: https://it.wordpress.org/plugins/woo-fattureincloud/

**Reply by valmoz** [reply]
Ok @logoso, grazie mille per il link.
WooFatture non è un plugin ufficiale Fatture in Cloud (anche se siamo costantemente in contatto con il loro team), quindi sfortunatamente non possiamo fornirle supporto direttamente.
Le consiglio di contattare direttamente il loro supporto sul sito https://woofatture.com/ loro le sapranno sicuramente dare una risposta in merito.

Credo comunque che il plugin richieda anche WooCommerce, mentre lei ha parlato solo di WP, forse è questo il problema?

Le chiederei comunque di tenerci aggiornati sullo stato della richiesta, in quanto potrebbe essere utile anche per altri utenti in futuro.

Grazie
Mauro
